projects/sensory_interface/public/app.js

```diff
@@ -20,29 +28,56 @@ function processData() {
         document.getElementById("tableContainer").removeChild(oldTable);
     }
     document.getElementById("tableContainer").appendChild(table);
-    addOnClickSoundToTable();
+    addOnClickAndTouchSoundToTable();
 }
 
 function fillRow(values, currentTr) {
     let myValue;
     for (myValue of values) {
-        let td = document.createElement("td");
+        let td = document.createElement("TD");
```

Comment:
If `table` and `tr` are lowercase, `td` should be as well, to stay consistent. Is there a reason you changed it?

Reply:
I double checked that, it's a lower case. This is strange!! I had a bug because the getTagName returned upper case when called on a td element.
Anyway, I reverted the change.

Reply:
Maybe `tagName` is uppercasing everything. In any case, you can do `tagName.toLowerCase()` before you compare, just to be sure.